codingstandards.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<ruleset name="TripalCultivate">
```

Comment:
I think I know where this name came from, but should we just name this "Tripal"?
```suggestion
<ruleset name="TripalCultivate">
```
```suggestion
<ruleset name="TripalCultivate">
```
```suggestion
<ruleset name="Tripal">
```